presto-docs/src/main/sphinx/sql/create-table.rst

```diff
@@ -9,7 +9,7 @@ Synopsis
 
     CREATE TABLE [ IF NOT EXISTS ]
     table_name (
-      column_name data_type [, ...]
+      (column_name data_type | LIKE existing_table_name [(INCLUDING | EXCLUDING) PROPERTIES]) [, ...]
```

Comment:
The documentation grammar should use curly braces rather than parentheses, because parentheses are used as literals in the grammar (they appear in SQL). Format like this:
```
table_name (
  { column_name data_type
  | LIKE existing_table_name [ { INCLUDING | EXCLUDING } PROPERTIES ] }
  [, ...]
)